pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -56,6 +57,7 @@
   private Set<String> _invertedIndexColumns = new HashSet<>();
   private Set<String> _rangeIndexColumns = new HashSet<>();
   private int _rangeIndexVersion = IndexingConfig.DEFAULT_RANGE_INDEX_VERSION;
+  private FSTType _fstTypeForFSTIndex = FSTType.LUCENE;
```

Comment:
(minor) Suggest renaming to `_fstIndexType`?

Reply:
There will be a FST type for text index as well, hence the disambiguation

Reply:
Understood. Suggest making it consistent with the getter and setter, and we can add another one for the text index FSTType if necessary in the future. We might also decide to use the same one for both text and fst.